src/consumer/consumer.js

```diff
@@ -42,9 +44,9 @@ function consumerPromise({id, init}) {
 			Error.undefinedValue('Consumer instance:: Worker Id undefined');
 		}
 
-		log.info(`[WORKER::${id}] Running consumer function.`);
-
 		async function messageHandler(msg) {
+			log.notice(`[CONSUMER::${id}] Received object.\
+				\r Running message handler`);
```

Comment:
Why `\r` and not `\n`?

Might even be better to move all the log messages to constants at the top of the file and then do something like:

    LOG_MESSAGE = `[CONSUMER::${id}] Received object.
    Running message handler`;